packages/apps/graph/src/scripts/devnet/get-balance.ts

```diff
@@ -0,0 +1,35 @@
+import { Pact } from '@kadena/client';
+import type { IPactDecimal } from '@kadena/types';
+
+import config from './config';
+import type { IAccount } from './helper';
+import { dirtyRead } from './helper';
+
+export async function getBalance(account: IAccount): Promise<IPactDecimal> {
+  console.log(`Getting balance for ${account.account}`);
```

Comment:
Instead of console.log, we could use a more generic solution. That can also be used to log to file etc. Maybe we can use a logging interface that, based on env vars or otherwise logs in different ways.

I made something 😅 😉 
https://github.com/kadena-community/kadena.js/blob/feat/ag/generic-logger/packages/apps/graph/src/utils/logger.ts